mlflow/utils/search_utils.py

```diff
@@ -187,7 +199,21 @@ def _invalid_statement_token(cls, token):
     def _process_statement(cls, statement):
         # check validity
         invalids = list(filter(cls._invalid_statement_token, statement.tokens))
-        if len(invalids) > 0:
+        if len(invalids) % 3 == 0 and len(invalids) != 0:
```

Comment:
Could you add a comment on what this is doing and why? I'm not sure why this PR requires this addition, but it seems complicated.

Reply:
This is a part of the code we will remove once @dbczumar PR will be released in sqlparse, before that it didn't recognize the LIKE keyword.